framework/Objects/Cronjob.php

```diff
@@ -0,0 +1,182 @@
+<?php
+
+namespace JustCoded\WP\Framework\Objects;
+
+/**
+ * Class Cronjob
+ *
+ * @package JustCoded\WP\Framework\Objects
+ *
+ * @method Cronjob instance() static
+ */
+abstract class Cronjob {
+	use Singleton;
+
+	/**
+	 * Constant for single cron.
+	 */
+	const FREQUENCY_ONCE = 'single';
+
+	/**
+	 * Constant for hourly cron.
+	 */
+	const FREQUENCY_HOURLY = 'hourly';
+
+	/**
+	 * Constant for twicedaily cron.
+	 */
+	const FREQUENCY_TWICEDAILY = 'twicedaily';
+
+	/**
+	 * Constant for daily cron.
+	 */
+	const FREQUENCY_DAILY = 'daily';
+
+	/**
+	 * Cron id.
+	 *
+	 * @var string
+	 */
+	protected $ID;
+
+	/**
+	 * Start time.
+	 *
+	 * @var int|string
+	 */
+	protected $start;
+
+	/**
+	 * Schedule name.
+	 *
+	 * @var string
+	 */
+	protected $schedule;
```

Comment:
let's rename to $frequency (same as constants) or $recurrence (same as parameter in WP function)